rueidis/rueidis.go

```diff
@@ -0,0 +1,113 @@
+package rueidis
+
+import (
+	"context"
+	"time"
+
+	"github.com/gofiber/fiber/v2/utils"
```

Comment:
why not to use github.com/gofiber/utils here

Reply:
True, will change it in a bit.

Reply:
Done, using `utils/v2` now